samples/boards/stm32/power_mgmt/serial_wakeup/boards/nucleo_wb55rg.overlay

```diff
@@ -1,19 +1,35 @@
 /*
  * Copyright (c) 2022 Linaro Limited
+ * Copyright (c) 2022 STMicroelectronics
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-&clk_hsi {
-	status = "okay";
+&cpu0{
+	/* USART Wakeup requires automatic HSI16 switch on in deepsleep mode
+	 * which isn't possible in Stop Mode 2.
+	 * Remove Stop Mode 2 from supported modes
+	 */
+	cpu-power-states = <&stop0 &stop1>;
```

Comment:
@ceolin For PM_DEVICE more, it would be great to have a way to specify that wakeup capability on a given device is possible only down to a certain state. Currently we need to disable the non supported state for the whole application, while the device wakeup source capability could be disabled dynamically by the application.